docs/pages/_app.js

```diff
@@ -1,10 +1,10 @@
 import "@csstools/normalize.css";
-import "@splitgraph/design/css/base.css";
+// import "@splitgraph/design/css/base.css";
 
 // Note: Ideally we could import sphinxtheme as a module in only the sphinx
 //       pages, but that requires upating the Sphinx -> mdx transform files
 //       to make sure that every className is correctly prefixed (todo later)
-import "@splitgraph/design/css/sphinxtheme.css";
+// import "@splitgraph/design/css/sphinxtheme.css";
```

Comment:
What do we do about these imports? Can we remove them?

Reply:
See the commit message. I'm not interested in deprecating these files in this branch. This is just to get typechecking passing so we can upgrade MUI, which is the goal of this branch.

Reply:
Right, I missed it, thanks for pointing it out